consensus/integration/blockordelay_test.go

```diff
@@ -38,19 +44,45 @@ func blockNodesFirstMessages(n uint64, denyList ...*Node) BlockOrDelayFunc {
 	}
 }
 
-func blockReceiverMessagesByPercentage(percent int) BlockOrDelayFunc {
-	rand.Seed(time.Now().UnixNano())
+// blockReceiverMessagesRandomly drops messages randomly with a probability of `dropProbability` ∈ [0,1]
+func blockReceiverMessagesRandomly(dropProbability float32) BlockOrDelayFunc {
+	lock := new(sync.Mutex)
+	prng := rand.New(rand.NewSource(64))
```

Comment:
Why 64? 🤔

Reply:
🤦 that is an artifact of my experimenting with initializing the random number generator. Fixed in https://github.com/onflow/flow-go/pull/3096/commits/2533b3e2aba7cd3b391781f21adb141e5a3e8ed3